File: arraycount.py
aaaa={}
count = ["ant","ant","ant","ant","ant","ant","ant","ant","ant","ant","ant","ant","a","b",'apple','w','wf']
# 리스트의 요소의 이름과 갯수를 딕셔너리화하기
for i in count:
    try: aaaa[i] += 1
    except: aaaa[i]=1

# 딕셔너리의 키를 리스트화하기 
co = list(aaaa.keys())

# 키와 밸류(갯수) 프린트
for c in co:
    print(c)
    print(aaaa.get(c))

File: arraycount.js
const count = ["ant","ant","ant","ant","ant","ant","ant","ant","ant","ant","ant","ant","a","b",'apple','w'];

// 배열 중복제거
const as = Array.from(new Set(count));

// 배열 중복제거한 리스트에서 하나씩 뽑아 개수 프린트
for (a of as){
console.log(a)
let co = count.filter(element => a === element).length;
console.log(co)};